pkgs/games/hex-a-hop/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib, stdenv, makeDesktopItem, copyDesktopItems, fetchurl, fetchzip, SDL, SDL_mixer, SDL_ttf }:
+
+stdenv.mkDerivation rec {
+  pname = "hex-a-hop";
+  version = "1.1.0";
+
+  src = fetchzip {
+    url = "https://downloads.sourceforge.net/project/hexahop/${version}/hex-a-hop-${version}.tar.gz";
+    sha256 = "sha256-fBSvNtgNR0aNofbvoYpM1e8ww4ARlXIvrQUvJqVGLlY=";
+  };
+
+  icon = fetchurl {
+    url = "https://sources.debian.org/data/main/h/hex-a-hop/${version}%2Bgit20140926-1.1/data/hex-a-hop.png";
+    sha256 = "sha256-Vh/1wwRmC2eSD/+mk1Oqt7EX4a4k++nUAbWQD2P2hNA=";
+  };
```

Comment:
Please just install this file from `${src}/data`.

Reply:
I can't check atm. but why does debian's source tree contain a different icon than the upstream tree? Where is it from?

Reply:
I don't know where it's from but it's similar. The source tree contains a bmp file that is cruder. The version from Deb ian is cleaned up (and a png).

Reply:
As I mentioned, it's in the source directory. https://sourceforge.net/p/hexahop/code/ci/master/tree/data/hex-a-hop.png

Reply:
Again, that's the master branch, not 1.1.0

Reply:
I'll change the link to those (with the sha not the branch though)

https://sourceforge.net/p/hexahop/code/ci/f0ba80de6da8fc073ab818d46361bf8867620ad7/tree/data/hex-a-hop.png
https://sourceforge.net/p/hexahop/code/ci/f0ba80de6da8fc073ab818d46361bf8867620ad7/tree/data/hex-a-hop.desktop

Reply:
Ah, interesting. Fetch it from dfa365a90be5c52ef21235a9e90a865b04da3ad4 from the source repo then and add a comment to remove it in the next version.

Reply:
Added both (the desktop one was from a later commit)